### DISCUSSION TITLE
PDF showing several pages in each page

### DISCUSSION THREAD

**Discussion by Ralstro** [sumatrapdfreader/sumatrapdf#3890] [Q&A] [answered]
Hi

I use an old Dell laptop to display music on my piano. I bought it because it was the only one that would open out 180 degrees and fit on my music stand. It is running Windows XP and I use Sumatra because it runs really well on a machine so old.

Some files take a little time to render but on the whole everything is fine except with one PDF. On this it displays the page as normal but the music shows three pages side by side over a black box. I suspect that the original file was in MOBI ebook format and has been converted.

I have attached a picture to show the fault. Every other PDF reader displays the file correctly but even Sumatra 3.4.6 on my Windows 10 computer will not display this properly.

Things I have already tried are optimising and flattening the PDF but neither has worked.
![Sumatra problem](https://github.com/sumatrapdfreader/sumatrapdf/assets/151367665/f697d54f-e070-44e6-8f01-e26e2ff016ef)

**Comment by GitHubRulesOK** [comment]
there is no way of knowing if its fixable without a sample but billions of pay this and that copies like this so no way to tell which one your using
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/96f25e31-90d3-43f3-9e64-cc9eece6b925)

**Reply by Ralstro** [reply]
Thank you for taking the time to get back to me.

I have uploaded the full PDF to see if you have the same outcome.
[Piano Solos for Every Occasion.pdf](https://github.com/sumatrapdfreader/sumatrapdf/files/13402938/Piano.Solos.for.Every.Occasion.pdf)


I'm sorry but I didn't understand the second part of your sentence about what I am using. If you mean Sumatra version it was the last one that supported XP.

Thanks again for your response, I really appreciate it.

Ralph.

**Comment by GitHubRulesOK** [comment, marked as answer]
It Fails to work in Adobe Acrobat thus I consider it is broken, and even a reprint from another route is visually good but with bad text underlay. 
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/aa2282d8-b650-468b-843e-0577841c93ff)


Well at $25 for 60 songs that are $5 each the best bet is find a second-hand original copy rather than a Calibre conversion of dubious means what about use an epub and a different conversion 
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/6a67176d-59b9-488f-82a6-147c3241363f)
avoid the PDF option go for the "Origional PDF" its an epub! https://vdoc.pub/download/piano-solos-for-every-occasion-the-complete-resource-for-all-pianists-1r3pkhsopo20

**Reply by Ralstro** [reply]
Thanks again for getting back to me. 

I am surprised that it didn't work in Adobe as it opens fine in PDF X-change Editor. I see now that it is a Calibre e-book but I'm not sure where I got it from.

Of course it doesn't matter now because I have the epub from your link!  I didn't even know that Sumatra could read the epub format, that is one thing that X-change cannot do, so thank you again for that.

I am hugely grateful for your help with this and want to thank you very much.

All best wishes, Ralph.

**Reply by GitHubRulesOK** [reply]
It is not just Acrobat or SumatraPDF have problems with the PDF variant, I tried others where it shows 3 pages per page. As far as I can tell its using an unconventional cropping frame around a multipage scanned image, so not sure why it works so well in PDFXEdit, but PDF is a very odd format in that respect.

**Reply by Ralstro** [reply]
Thanks, it is an odd format.

I have scanned every music book I own into PDF and it has been completely successful. I can't tell you what a difference it has made. Before I bought the laptop so many books wouldn't lie flat and had to be clipped open so that I could play them. Then they had to be unclipped to turn the page, it was so frustrating!  The use of PDFs is so much better as the pages turn with one click of a button. 

The books I have scanned myself all appear as a complete page. Some PDFs I have found online appear in three stages as if they are three layers. The blank page appears followed by a 'ghost' image of the music, then the same image much darker. I have no idea why this happens as a scan is just one bitmap.

The only other time I have noticed this is when I export Corel-draw designs to PDF. Then every separate component of the drawing appears until the image is complete.

Other than that I find PDF to be perfect for my needs. 

Sorry to have taken up so much of your time when you have been so kind!
